src/EditorFeatures/CSharpTest/GenerateFromMembers/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.cs

```diff
@@ -1052,7 +1052,7 @@ public override bool Equals(object obj)
     {
         var foo = obj as Foo;
         return foo != null &&
```

Comment:
📝 This should ideally be generated as `foo is object`, but obviously that should be a separate pull request.

Reply:
https://github.com/dotnet/roslyn/issues/39253